internal/commands/resolve.go

```diff
@@ -105,14 +159,41 @@ func ResolveIncidentByDialog(ctx context.Context, client bot.Client, logger log.
 	return nil
 }
 
-func createResolveChannelAttachment(inc model.Incident, userName string) slack.Attachment {
+func setMeetingDate(d *time.Time, postMortemGapDays int, timezone string) (string, string) {
+	previewPostMortemDate := d.AddDate(0, 0, postMortemGapDays)
+
+	switch previewPostMortemDate.Weekday() {
+	case time.Saturday:
+		postMortemGapDays += 2
+	case time.Sunday:
+		postMortemGapDays++
+	default:
+		break
+	}
+
+	utc, _ := time.LoadLocation(timezone)
```

Comment:
Just remembering, this error ignored will be handled in v2

Reply:
Even if the proper handling will be done in the future, you can return in a `err` variable, and at least return it, or even log it.